app/views/tasks/_task.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr>
+  <td class="p-0 text-center">
+    <div class="custom-checkbox custom-control">
+      <input type="checkbox" data-checkboxes="mygroup" class="custom-control-input" id="checkbox-1">
+      <label for="checkbox-1" class="custom-control-label">&nbsp;</label>
+    </div>
+  </td>
+  <td class="col-2"><%= task.title %></td>
+  <td class="col-3"><%= task.discription %></td>
```

Comment:
fix spells

Reply:
it will fix by change the field name. just changing

Reply:
done